gr-uhd/3.8: RX channel .... out of range for configured RX frontends

## Issue Description ##
Using two channels with the UHD USRP Source block will throw an error
```
RX channel ... out of range for configured RX frontends
```
![gr38_b210_2ch](https://user-images.githubusercontent.com/11846824/61090343-1834ba80-a3f3-11e9-8d15-022b188e96ea.png)


## Setup Details ##
- UHD 3.14.1.0
- GR master 3.8 @ https://github.com/gnuradio/gnuradio/commit/cff05c36dfc3908c3933773ac789959050fae45e
- USRP B210

## Expected Behavior ##
- No error

## Actual Behavior ##

```
Generating: '/home/user/top_block.py'

Executing: /usr/bin/python3 -u /home/user/top_block.py

[0;32m[INFO] [UHD] [0;39mlinux; GNU C++ version 7.3.0; Boost_106501; UHD_3.14.1.HEAD-0-gbfb9c1c7
[0;31m[ERROR] [UHD] [0;39mDevice discovery error: send_to: Operation not permitted
[0;31m[ERROR] [USRP2] [0;39mUSRP2 Network discovery error send_to: Operation not permitted
[0;31m[ERROR] [X300] [0;39mX300 Network discovery error send_to: Operation not permitted
[0;31m[ERROR] [UHD] [0;39mDevice discovery error: send_to: Operation not permitted
[0;32m[INFO] [B200] [0;39mDetected Device: B210
[0;32m[INFO] [B200] [0;39mOperating over USB 3.
[0;32m[INFO] [B200] [0;39mDetecting internal GPSDO.... 
[0;32m[INFO] [GPS] [0;39mFound an internal GPSDO: LC_XO, Firmware Rev 0.929a
[0;32m[INFO] [B200] [0;39mInitialize CODEC control...
[0;32m[INFO] [B200] [0;39mInitialize Radio control...
[0;32m[INFO] [B200] [0;39mPerforming register loopback test... 
[0;32m[INFO] [B200] [0;39mRegister loopback test passed
[0;32m[INFO] [B200] [0;39mPerforming register loopback test... 
[0;32m[INFO] [B200] [0;39mRegister loopback test passed
[0;32m[INFO] [B200] [0;39mSetting master clock rate selection to 'automatic'.
[0;32m[INFO] [B200] [0;39mAsking for clock rate 16.000000 MHz... 
[0;32m[INFO] [B200] [0;39mActually got clock rate 16.000000 MHz.
Traceback (most recent call last):
  File "/home/user/top_block.py", line 152, in <module>
    main()
  File "/home/user/top_block.py", line 130, in main
    tb = top_block_cls()
  File "/home/user/top_block.py", line 90, in __init__
    self.uhd_usrp_source_0.set_center_freq(98e6, 1)
  File "/home/user/sdr/installs/v3.14.1.0_master/lib/python3.6/dist-packages/gnuradio/uhd/uhd_swig.py", line 3599, in set_center_freq
    return _uhd_swig.usrp_source_sptr_set_center_freq(self, *args)
RuntimeError: LookupError: IndexError: multi_usrp: RX channel 35015184 out of range for configured RX frontends

>>> Done (return code 1)
```

## Steps to reproduce the problem ##
- Run attached flowgraph

## Additional Information ##
[b210_2ch.zip](https://github.com/gnuradio/gnuradio/files/3384252/b210_2ch.zip)

--- a/gr-uhd/grc/gen_uhd_usrp_blocks.py
+++ b/gr-uhd/grc/gen_uhd_usrp_blocks.py
@@ -144,10 +144,10 @@ templates:
                 ${'%'} if stream_args:
                 args=${'$'}{stream_args},
                 ${'%'} endif
-                ${'%'} if stream_chans:
+                ${'%'} if eval(stream_chans):
                 channels=${'$'}{stream_chans},
                 ${'%'} else:
-                channels=range(${'$'}{nchan}),
+                channels=list(range(0,${'$'}{nchan})),
                 ${'%'} endif
             ),
             ${'%'} if len_tag_name:

